pkgs/by-name/ap/apple-sdk/scripts/lock-sdk-deps.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env nix-shell
```

Comment:
Seems like a lot of effort for just these two things, especially since it would have to work around the `by-name` limitation on not referencing paths outside the package’s location.